### PR TITLE
fix: base account triggers coinbase wallet extension connection

### PR DESCRIPTION
**Pull request by 0xmkh**
# Description

Fixed an issue where selecting Base Account incorrectly triggered the Coinbase Wallet extension connection instead of the Base Account SDK connection

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-4267

# Checklist

- [x] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections
- [x] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

---

> [!NOTE]
> Prioritizes Base Account over Coinbase during connector resolution to avoid misrouting to the Coinbase extension; adds unit tests and relaxes Chromatic CI to not fail on visual diffs.
> 
> - **Connectors/Resolution**
>   - Add `ConnectorUtil.getPriority` and `sortConnectorsByPriority` to rank connectors (Base Account > Coinbase/SDK > others).
>   - Use prioritized sorting in `ConnectorController.getConnectorById` and `getConnector` to select the correct connector.
> - **Tests**
>   - Add unit tests for `sortConnectorsByPriority` covering Base Account, Coinbase/SDK ordering, stability, and edge cases.
> - **CI**
>   - Update Chromatic workflow: set `exitZeroOnChanges: true` and remove failing step so visual diffs don’t fail the workflow.
> - **Release**
>   - Patch bumps across multiple `@reown/*` packages with changelog entry for the fix.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 6d41ef48b1756068eec8d23c21957145af5083b9. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

File: packages/controllers/src/utils/ConnectorUtil.ts [hunk 1]
```diff
@@ -181,6 +181,38 @@ export const ConnectorUtil = {
       return 0
     })
   },
+
+  /**
+   * Returns the priority of a connector.
+   * @param connector - The connector to get the priority of.
+   * @returns The priority of the connector.
+   */
+  getPriority(connector: ConnectorWithProviders) {
```
Comment (enesozturk) [inline packages/controllers/src/utils/ConnectorUtil.ts:194]: Is this 'priority' only for Coinbase/Base wallets? If so can you add description for it? It's not understandable why we have this Priority concept needed
